Tests.Integration.Presentation.Web/Internal/GdprReport/GdprReportApiV2Test.cs

```diff
@@ -0,0 +1,61 @@
+﻿using Core.DomainModel.Organization;
+using Core.DomainModel;
+using Presentation.Web.Models.API.V1;
+using System.Threading.Tasks;
+using Tests.Integration.Presentation.Web.Tools;
+using Tests.Toolkit.Patterns;
+using Xunit;
+using Tests.Integration.Presentation.Web.Tools.Internal;
+using System.Collections.Generic;
+using Presentation.Web.Models.API.V2.Internal.Response;
+using System;
+using Tests.Integration.Presentation.Web.Tools.External;
+using Presentation.Web.Models.API.V2.Request.System.Regular;
+using Presentation.Web.Models.API.V2.Request.SystemUsage;
+using Presentation.Web.Models.API.V2.Response.SystemUsage;
+
+namespace Tests.Integration.Presentation.Web.Internal.GdprReport
+{
+    public class GdprReportApiV2Test : WithAutoFixture
+    {
+
+        [Fact]
+        public async Task Can_Get_Gdpr_Report()
+        {
+            var org = await CreateOrganizationAsync();
+            await CreateUsage(org.Uuid);
+
+            var response = await GdprReportV2Helper.GetGdprReportAsync(org.Uuid);
+
+            Assert.True(response.IsSuccessStatusCode);
+            var body = await response.ReadResponseBodyAsAsync<IEnumerable<GdprReportResponseDTO>>();
+            Assert.NotEmpty(body);
```

Comment:
Since the test creates a test usage above, can it assert that something related to this particular usage exists in the response?